app/js/users/userModel.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * userModel.js
+ * Stores state related to a particular user
+ */
+
+function userModel() {
+    return {
+        // State
+        Email: '',
+
+        // Event handlers
+        handlers: {
+          'listUsers': 'listUsers'
+        },
+
+        listUsers: function (r) {
+            this.users = r.response;
```

Comment:
Even though I think will behave correctly, we should list all our state attributes above (like with `Email`)